addon/plugins/article-structure-plugin/utils/structure.ts

```diff
@@ -38,26 +45,24 @@ export function constructStructureNodeSpec(config: {
     },
     allowSplitByTable: config.allowSplitByTable,
     toDOM(node) {
-      return [
-        'div',
-        {
-          property: node.attrs.property as string,
-          typeof: node.attrs.typeof as string,
-          resource: node.attrs.resource as string,
+      return renderRdfaAware({
+        renderable: node,
+        tag: 'div',
+        attrs: {
+          ...node.attrs,
+          class: 'say-editable',
         },
-        0,
-      ];
+        content: 0,
+      });
     },
     parseDOM: [
       {
         tag: 'div',
+        priority: 55,
```

Comment:
Could you add a comment as to why this priority is added/needed?

Reply:
I removed this one actually as it wasn't necessary...